src/main/java/com/ksm/wstbackoffice/controller/CountryController.java

```diff
@@ -44,4 +47,20 @@ public CountryController(CountryService countryService) {
         return ResponseEntity.status(HttpStatus.OK)
                 .body(country);
     }
+
+    @PostMapping("/start-with")
```

Comment:
We were talking a lot about REST best practices.
Naming the URI `/start-with` you violate the `resource-based URI` principle of the Richardson Maturity Model.
And we learned the rules definitely not to violate them.
Can you find a better solution here?

Reply:
done